cmd/share.go

```diff
@@ -63,8 +67,21 @@ var shareCmd = &cobra.Command{
 			fmt.Println("Share revoked for client " + refereeClientID)
 		} else {
 			expiration, _ := cmd.Flags().GetInt64("expiration-seconds")
+			availableAfter := time.Now()
+
+			availableAfterInput, _ := cmd.Flags().GetString("available-after")
```

Comment:
Check the error in case the input is invalid

Reply:
fixed